IdentityCore/src/webview/embeddedWebview/challangeHandlers/MSIDPKeyAuthHandler.m

```diff
@@ -72,6 +73,23 @@ + (BOOL)handleChallenge:(NSString *)challengeUrl
     NSMutableURLRequest *responseReq = [[NSMutableURLRequest alloc] initWithURL:responseUrlComp.URL];
     [responseReq setValue:kMSIDPKeyAuthHeaderVersion forHTTPHeaderField:kMSIDPKeyAuthHeader];
     [responseReq setValue:authHeader forHTTPHeaderField:MSID_OAUTH2_AUTHORIZATION];
+
+    // Adding refreshTokenCredential (PRT) header to the challenge response. Header is available in customheaders dict
+    BOOL refreshTokenIsValid = NO;
+    if (customHeaders != nil && customHeaders.count > 0)
```

Comment:
nit: nil check doesn't need to be done in obj c. Message passed to a nil object doesn't throw an error, it translates to 0/ false. Can simplify the condition as:  if (!customeHeaders.count)
Though, I don't think you need to check the count here either